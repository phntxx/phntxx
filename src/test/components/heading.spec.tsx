import { render } from "@testing-library/react";
import Heading from "../../components/heading";

describe("heading.tsx", () => {
  it("should render component without label without crashing", () => {
    const { asFragment } = render(<Heading />);
    expect(asFragment).toMatchSnapshot();
  });
});
