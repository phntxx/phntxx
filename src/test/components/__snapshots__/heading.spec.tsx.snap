// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`heading.tsx should render component without label without crashing 1`] = `[Function]`;
